data/man/meson.build

```diff
@@ -1 +1,25 @@
-install_man('tilix')
\ No newline at end of file
+install_man('tilix.1')
+
+po4a = find_program('po4a-translate', required: false)
+
+if po4a.found()
+
+    locales = [ 'ca', 'cs', 'de', 'en_GB', 'es', 'fr', 'hr',
+                'it', 'nb_NO', 'nl', 'oc', 'pl', 'pt_BR', 'pt',
+                'pt_PT', 'ro', 'ru', 'sr', 'tr', 'uk', 'zh_Hant' ]
+
+    foreach locale : locales
+        custom_target('man_' + locale,
+            output: locale,
+            input: [ 'tilix.1', 'po/' + locale + '.man.po' ],
+            command: [ po4a, '--keep', '0',
+                             '--format', 'man',
+                             '--master', '@INPUT0@',
+                             '--po', '@INPUT1@',
+                             '--localized', '@OUTPUT@/man1/tilix.1'],
+            install: true,
+            install_dir: get_option('mandir')
```

Comment:
Needs to have `man1` specified at the end of the mandir, since the option is the base directory and `install_man()` adds a bit of magic on top of that to select the correct subdirectory (including for localized output filenames).

Unfortunately, install_man cannot accept as input, a built file from custom_target. And my efforts to add an API for custom_target functionality inside install_man, have stalled. :(

Oh well, one day I'll surely figure out something that makes everyone happy.